doc/user/separate.xml

```diff
@@ -44,115 +44,49 @@
-notice that all the builds are done using source files and object files from
-the build directory. For example, F<build/world/world.o> is compiled from
-F<build/world/world.c>, and F<export/include/world.h> is installed from
-F<build/world/world.h>. This is accomplished on most systems by the simple
-expedient of ``hard'' linking the required files from each source directory
-into the appropriate build directory.
-
-The links are maintained correctly by Cons, no matter what you do to the
-source directory. If you modify a source file, your editor may do this ``in
-place'' or it may rename it first and create a new file. In the latter case,
-any hard link will be lost. Cons will detect this condition the next time
-the source file is needed, and will relink it appropriately.
-
-You'll also notice, by the way, that B<no> changes were required to the
-underlying F<Conscript> files. And we can go further, as we shall see in the
-next section.
-
-=head2 Explicit references to the source directory
-
-When using the C<Link> command on some operating systems or with some
-tool chains, it's sometimes useful to have a command actually use
-the path name to the source directory, not the build directory.  For
-example, on systems that must copy, not "hard link," the F<src/> and
-F<build/> copies of C<Linked> files, using the F<src/> path of a file
-name might make an editor aware that a syntax error must be fixed in the
-source directory, not the build directory.
-
-You can tell Cons that you want to use the "source path" for a file by
-preceding the file name with a ``!'' (exclamation point).  For example,
-if we add a ``!'' to the beginning of a source file:
-
-  Program $env "foo", "!foo.c";	# Notice initial ! on foo.c
-
-Cons will compile the target as follows:
-
-  cc -c src/foo.c -o build/foo.o
-  cc -o build/foo build/foo.o
-
-Notice that Cons has compiled the program from the the F<src/foo.c>
-source file.  Without the initial ``!'', Cons would have compiled the
-program using the F<build/foo.c> path name.
+  It's often useful to keep any built files completely
+  separate from the source files. Consider if you have a
+  project to build software for a variety of different
+  controller hardware.  The boards are able to share a
+  lot of code, so it makes sense to keep them in the same
+  source tree, but certain build options in the source code
+  and header files differ.  If you build "Controller A" first,
+  then "Controller B", on the second build everything would
+  have to be rebuilt, because &SCons; sees that the build
+  instructions differ, and thus the targets that depend on those
+  different instructions are not valid for the current build.
+  Now when you go back and build for "Controller A",
+  things have to be rebuilt from scratch again for the same reason.
+  However, if you can separate the places the output files
+  go, this problem can be avoided.
+  You can even set up to do both builds in one invocation of &SCons;.
```

Comment:
I kinda prefer the verbosity of the commented out text. Though is it a duplicate of other info elsewhere at this point?

Reply:
You want to bring in all of the stuff from Cons?

My effort, since this is UGuide material, was to be a little more helpful, and less technically detailed, we can add some of this to manpage if that's needed?